.github/workflows/node.js.yaml

```diff
@@ -0,0 +1,66 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
```

Comment:
Because this is only going to use one version of node, I'd skip the matrix, and just install node 18 below and use that throughout.